### DISCUSSION TITLE
Task 'wrapper' not found in project ':FtcRobotController'.

### DISCUSSION THREAD

**Discussion by AryaCherukuri** [FIRST-Tech-Challenge/FtcRobotController#198] [Q&A] [answered]
This works for my friends but I am running into this issue, 

Task 'wrapper' not found in project ':FtcRobotController'.

* Try:
Run gradle tasks to get a list of available tasks. Run with --stacktrace option to get the stack trace. Run with --info or --debug option to get more log output. Run with --scan to get full insights.

I tried all suggestion I could find online and also tried different version of Android studio no help.
Not sure what I am doing worng here.

**Comment by cmacfarl** [comment]
I'm not sure there is one single answer to your question.

Google and this answer on StackOverflow seems to indicate that there can be multiple causes.

https://stackoverflow.com/questions/25172006/android-studio-build-fails-with-task-not-found-in-root-project-myproject

In order to more fully answer the question, we'd have to have more insight into your environment, how you set up your project, what the state of Android Studio is on your machine, etc...  For questions like this, the more information you can provide, the better.

**Reply by AryaCherukuri** [reply, marked as answer]
for me the issue was project was failing to find gradle-wrapper.properties file. I got the issue resolved by pointing to the location. 
Then next time when I opend this issue was gone.
![image](https://user-images.githubusercontent.com/67491274/137636392-74db25f4-778f-4e74-9049-7b66bb06c85c.png)
